platform/podjs/README.md

```diff
@@ -20,21 +20,160 @@ feature. While we could support the creation of a container and a
+    setup for `src`, which in this case it's the `"node_modules/@polypoly-eu/podjs/dist/pod.js"`, and `dest` where it will be ended to (e.g. `dist`).
+    So, it looks like:
+
+```js
+   import copy from "@polypoly-eu/rollup-plugin-copy-watch";
+
+   export default {
+      input: "src/your-input-file.js",    // point to your source file
+      output: {
+         file: "dist/myfile.js", // point to your output file
+         format: "iife",     // the format of your output file, for now only iife is supported
+         globals: {
+               // where you define all the external packages you want to use
+               pod: "pod",
+            },
+         }
+      },
+      plugins: [
+         ...
+         copy({
+            targets: [
+            {
+               src: [
+                  "node_modules/@polypoly-eu/podjs/dist/pod.js",
+                  ...
+               ],
+               dest: "dist",
+            },
+         ]
+         }),
+      ]
+   }
+```
+
+This way you have copied successfully the `pod.js` file from the `@polypoly-eu/podjs` module, and you can use it in your feature.
+
+4.  So, to compile the input files and run the previous configurations,
+    you should define a `build` script in the `package.json` file,
+    which will run the `rollup` command.
+
+    For example, for Javascript features, you can use:
+
+```json
+   {
+      ...
+      "scripts": {
+         "build": "shx rm -rf dist && rollup -c",
+         ...
+      },
+   }
+```
+
+Or for Typescript features, you will also need to compile with `tsc`:
+
+```json
+   {
+      ...
+      "scripts": {
+         "build": "shx rm -rf dist && tsc && rollup -c",
+         ...
+      },
+   }
+```
+
+5.  You are ready to build your feature. Try:
+
+    `$ npm run build`
+
+    and you should have a `dist` folder with your feature included the `pod.js` file.
+
+6.  Now, the second important step is to include the `pod.js` from your feature's main document, before the feature's own code.
+    So, in your feature's `HTML` file, you should along with the rest polyPod implementations make sure you include `pod.js` script, like this:
+
+```html
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <script src="pod.js"></script>
+        ...
+    </head>
+</html>
+```
+
+7.  To verify that everything is in place, you can test it by following these steps:
 
-## Usage
+    -   Open your feature's `HTML` file in your browser, check that `URL`s are supported without the need to run an `HTTP server`.
+    -   Make sure your manifest is exposed as window and that there's the correct navigation bar colour and the correct (localised) feature name.
+    -   Check there's the correct backwards navigation within the feature and that it works successfully via the native back button of the browser.
```

Comment:
Also framed like test instructions, we can just point out that back navigation works via the native back functionality of the browser. We could also point out that it's worth taking a look at any console logs for further navigation instructions.